src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -17,17 +18,28 @@ export enum MarkdownEditorTabs {
 
 export type MarkdownEditorProps = {
   placeholder?: string
+  forumId?: string
+  editTitle?: string
+  editText?: string
+  onSave: (text: string, title?: string) => void
+  onCancel: () => void
```

Comment:
Let's also make this fully controlled component:
```suggestion
  text?: string
  setText?: (text: string) => void
```